ECMA2Yaml/ECMAHelper/YamlConverters/SDPYamlConverter.Member.cs

```diff
@@ -23,7 +23,7 @@ public MemberSDPModel FormatSingleMember(Member m)
 
             var knowTypeParams = m.Parent.TypeParameters.ConcatList(m.TypeParameters);
 
-            if (m.ReturnValueType != null && m.ItemType != ItemType.Event)
+            if (m.ReturnValueType != null)
             {
                 var returns = m.ReturnValueType;
 
```

Comment:
I actually think the first part of the `if` statement below can be removed entirely:
```csharp
                if (returns.VersionedTypes.Count() == 1 )
                {
                    var oneReturn = returns.VersionedTypes.First();
                    if (oneReturn != null
                    && !string.IsNullOrEmpty(oneReturn.Value)
                    && oneReturn.Value != "System.Void")
                    {
                        sdpMember.Returns = new TypeReference()
                        {
                            Description = returns.Description,
                            Type = SDPYamlConverter.TypeStringToTypeMDString(oneReturn.Value, _store)
                        };
                    }
                }
```
because we don't use `.Returns` anymore, we always use `.ReturnsWithMoniker`.